frontend/src/metabase/visualizations/visualizations/PivotTable/PivotTable.tsx

```diff
@@ -77,6 +79,7 @@ function PivotTable({
   data,
   settings,
   width,
+  height: heightProp,
```

Comment:
Can we keep it as `height`?

Reply:
There's another `height` param that's provided by the `AutoSizer` component

`height={(isDashboard ? height : queryBuilderGridHeight) - scrollBarOffsetSize()} `